packages/e2e/scripts/local.runner.js

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+const Nightwatch = require('nightwatch');
+const browserstack = require('browserstack-local');
+let bs_local;
+
+try {
+
+  process.mainModule.filename = "./node_modules/nightwatch/bin/nightwatch"
+  // Code to start browserstack local before start of test
+  console.log("Connecting local");
+  Nightwatch.bs_local = bs_local = new browserstack.Local();
+  bs_local.start({'key': process.env.BROWSERSTACK_ACCESS_KEY }, (error) => {
+    if (error) throw error;
+
+    console.log('Connected. Now testing...');
+    Nightwatch.cli(function(argv) {
+      Nightwatch.CliRunner(argv)
+        .setup(null, () => {
+          // Code to stop browserstack local after end of parallel test
+          bs_local.stop(() => {});
+        })
```

Comment:
Same anonymous function is used twice. Consider extracting it as a named function and `.setup(null, someFunc)`

Reply:
You don't agree?